osg_configure/configure_modules/gratia.py

```diff
@@ -618,89 +426,72 @@ def _configure_htcondor_ce_probe(self):
 
-        config_location = GRATIA_CONFIG_FILES['condor']
+        config_location = GRATIA_CONFIG_FILES['htcondor-ce']
         contents = open(config_location, "r", encoding="latin-1").read()
         re_obj = re.compile(r'(?m)^\s*DataFolder\s*=(.*)\s*$')
         match = re_obj.search(contents)
-        if match is not None:
-            data_folder = match.group(1)
-            data_folder = data_folder.strip('" \t')
-            # PER_JOB_HISTORY_DIR comes from the schedd, so if condor's not
-            # running, we can't get a value (SOFTWARE-1564)
-            history_dir = self._get_condor_history_dir(condor_config_val_bin)
-            if not history_dir:
-                self.log("Could not verify DataFolder correctness: unable to get PER_JOB_HISTORY_DIR. "
-                         "This may be caused by the condor schedd not running, or by PER_JOB_HISTORY_DIR "
-                         "not being defined.", level=logging.WARNING)
-            else:
-                # os.path.samefile will die if the paths don't exist so check that explicitly (SOFTWARE-1735)
-                if not os.path.exists(data_folder):
-                    self.log("DataFolder setting in %s (%s) points to a nonexistant location" % (
-                    config_location, data_folder),
-                             level=logging.ERROR)
-                    valid = False
-                elif not os.path.exists(history_dir):
-                    self.log("Condor PER_JOB_HISTORY_DIR %s points to a nonexistant location" % history_dir,
-                             level=logging.ERROR)
-                    valid = False
-                else:
-                    try:
-                        if not os.path.samefile(data_folder, history_dir):
-                            self.log("DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s "
-                                     "do not match, these settings must match!" % (config_location,
-                                                                                   data_folder,
-                                                                                   history_dir),
-                                     level=logging.ERROR)
-                            valid = False
-                    except OSError as e:
-                        self.log("Error comparing DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s:\n%s"
-                                 % (config_location, data_folder, history_dir, e),
-                                 level=logging.ERROR)
-                        valid = False
-
-            # Per Gratia-126 DataFolder must end in / otherwise gratia won't find certinfo files
-            if not data_folder.endswith('/'):
-                self.log("DataFolder setting in %s must end in a /" % config_location,
-                         level=logging.ERROR)
-                valid = False
+        if not match:
+            return True
```

Comment:
So if we can't find `DataFolder` in the `ProbeConfig` we assume that it's ok? I think we should complain loudly if that is somehow the case.

Reply:
Good point.